Bottom: bd9ad682dd3e18c7c11001e4be9b3456d52189c0
Top:    80e9fa8bd261c0f56eace7b953a020aafc81ef1e
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-20 13:58:00 +0800

Refresh of tun-introduce-0

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index ff14c99..8df315f 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -2613,7 +2613,7 @@ struct socket *tun_get_socket(struct file *file)
 }
 EXPORT_SYMBOL_GPL(tun_get_socket);
 
-struct skb_array *tap_get_skb_array(struct file *file)
+struct skb_array *tun_get_skb_array(struct file *file)
 {
        struct tun_file *tfile;
